spec/features/poll_spec.rb

```diff
@@ -157,14 +157,35 @@
     current_page.visit_page
 
     expect(current_page).to have_button_for_vote
+    expect(current_page).not_to have_link_for_results
```

Comment:
And this is why you have fragile tests. The rule of thumb goes like this: never use `not_to` in our tests, always rephrase it somehow. (`expect.not_to have_css` can be transformed into `expect.to have_no_css` and so on).

Those `have_nothing` finders are blocking; they will wait. The `.not_to have_something` variants will not.
More on this here: https://robots.thoughtbot.com/write-reliable-asynchronous-integration-tests-with-capybara